### PR TITLE
Expand Transform Feedback test

### PR DESCRIPTION
**Pull request by toji**
Verifies basic draw call behavior and varying queries.

### CODE REVIEW

**Comment by zhenyao** [comment]
LGTM

File: sdk/tests/js/webgl-test-utils.js [hunk 1]
```diff
@@ -1282,6 +1357,37 @@ var checkAreaInAndOut = function(gl, x, y, width, height, innerColor, outerColor
...
 
 /**
+ * Checks that an entire buffer matches the floating point values provided.
+ * (WebGL 2.0 only)
+ * @param {!WebGL2RenderingContext} gl The WebGL2RenderingContext to use.
+ * @param {number} target The buffer target to bind to.
+ * @param {!Array.<number>} expected The values expected.
+ * @param {string} opt_msg Optional. Message to associate with success. Eg ("should be red").
+ * @param {number} opt_errorRange Optional. Acceptable error in
+ *        color checking. 0 by default.
```
Comment (kenrussell) [inline sdk/tests/js/webgl-test-utils.js]: Docs need to be updated. "Acceptable error in comparisons. 0.001 by default."

**Comment by kenrussell** [comment]
Sorry for slow review. Could you please follow this up with the doc clarification? Doesn't need to be reviewed. Nice test.

**Comment by toji** [comment]
Documentation fixed in 18dba7c5

**Pull request merged**